pose_est/RgbdImages.py

```diff
@@ -656,6 +702,7 @@ def _fit_kinematics_model(self, model_name, color_list):
                 self.marker_sets[idx].markers[count].pos[:2] = markers_local
             count += 1
             if count == list_nb_markers[idx]:
+                ### Should this happen ??

```

Comment:
May be we can just remove every draw markers on the crop images and just do it on the full image when we save the markers.

Reply:
Also may be we do not need to return the "color_list" but the full image instead and maybe the markers positions (or the dict that we save).